lib/http_clients/underwriter.ex

```diff
@@ -110,28 +110,19 @@ defmodule HttpClients.Underwriter do
 
     partner = %Partner{partner_type: partner["partner_type"], slug: partner["slug"]}
 
-    credit_analysis = %CreditAnalysis{
-      warranty_region_status: credit_analysis["warranty_region_status"],
-      warranty_type_status: credit_analysis["warranty_type_status"],
-      warranty_value_status: credit_analysis["warranty_value_status"],
-      pre_qualified: credit_analysis["pre_qualified"]
-    }
-
-    proposal_simulation = %ProposalSimulation{
-      financing_type: proposal_simulation["financing_type"],
-      loan_requested_amount: proposal_simulation["loan_requested_amount"]
-    }
-
     %Proposal{
+      id: proposal["id"],
       sales_stage: proposal["sales_stage"],
+      financing_type: proposal_simulation["financing_type"],
+      loan_requested_amount: proposal_simulation["loan_requested_amount"],
       lost_reason: proposal["lost_reason"],
-      id: proposal["id"],
-      status: proposal["status"],
-      blearning_lead_score: proposal["blearning_lead_score"],
+      lead_score: proposal["blearning_lead_score"],
```

Comment:
```suggestion
      lead_score: proposal["lead_score"],
```
já que vamos mudar no underwriter tb